EIPS/eip-5489.md

```diff
@@ -0,0 +1,160 @@
+
+## Motivation
+NFTs have entered mainstream consciousness in a big way as NFT trading volume spiked 21,000% to more than $17 billion in 2021. Despite NFTs’ monumental rise as blockchain-based digital assets, we are just starting to scratch the surface of what is possible with NFTs.
+
+The emergence of NFTs brings more possibilities for creators and Web3 projects to build up decentralized communities, which also shifts the previous company-consumer relationship to more of a partnership. Each NFT holder is entitled to trade, promote, or create based on their NFTs to make profits. 
+
+However, there hasn’t been an effective approach to providing avenues of continued monetization for community holders. Going through all the NFT-related EIPs, the solutions to "making NFTs profitable" are limited to transfers ([EIP-721](./eip-721.md)), rental and mortgage ([EIP-4907](./eip-4907.md)).
+
+Inspired by hyperlink, the essence of World Wide Web, the proposed hNFT standard positions NFT as the medium for Web 3.0. The hNFT standard endows NFTs with a hyperlink extension so that it is possible to exercise the `fructus` right, meanwhile, keeps the `abusus`. With hyperlink extension, NFT owners can earn profit by authorizing any hyperlink slot to an address.
+
+To establish standardization for NFT hyperlinks, it is important to share a unified standard so as to facilitate collaboration among all applications.
+
+Right to permission access to information—could be composed with “usus,” “abusus,” and “fructus” to create a nuanced constellation of access rights.
+
+## Specification
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119.
+
+### NFT hyperlink extension
+
+NFT hyperlink extension extends EIP-721 with a `mapping<TokenId, mapping<SlotManagerAddress, uri>>`. The mapping means `TokenId` identified NFT has authorized `SlotManagerAddress` to manage the `uri` which is identified by `SlotManagerAddress`.
+
+Besides, extension also provide the supporting facilities, such as
+1. authorizeSlot/revokeSlot call pair used to manage slot authorization, and SlotAuthorizationCreate/Revoked event pair used to announce authorization management happens.
+2. setSlotUri/getSlotUri call pair used to manage/view uri on slot, and SlotUriUpdated used to announce uri management happens.
+
+### New URI format
+
+This extension tries to add a new URI format, which is prefixed with "hnft://". This URI format enables redirect or connect activity between hNFTs, it enables great imagination.
```

Comment:
```suggestion
This EIP adds a new URI scheme, "hnft://". This URI scheme enables redirecting or connecting activity between hNFTs.
```
Required: Fix grammar.
Required: Use "this EIP".
Required: Use correct terminology.
Recommended: Rephrasing.